src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final ColorSupplier COLOR_SUPPLIER = new ColorSupplier();
+    private static final int MAX_NUMBER = 101;
```

Comment:
Minor, keep in mind for upcoming tasks: The real MAX_NUMBER is 100, it should be incremented on `nextInt()` call